db/schema.rb

```diff
@@ -128,23 +128,38 @@
     t.index ["assay_attachment_id"], name: "index_assay_files_on_assay_attachment_id", using: :btree
   end
 
+  create_table "autocomplete_values", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_520_ci" do |t|
+    t.string   "field_name"
+    t.string   "value"
+    t.integer  "institution_id", null: false
+    t.datetime "created_at",     null: false
+    t.datetime "updated_at",     null: false
+    t.index ["field_name", "value", "institution_id"], name: "autocomplete_index", unique: true, using: :btree
+    t.index ["institution_id"], name: "index_autocomplete_values_on_institution_id", using: :btree
+  end
+
```

Comment:
**note**: I think this table doesn't exist anymore (but we might want to reintroduce it to speed up the autocomplete queries).

```suggestion
```

Reply:
Given that now autocomplete apply only to batches fields and the number of batches that each institution will have (in the order of few dozens, not like samples that there are a lot, hundreds or thousands) we don't want to reintroduce this table.